.github/workflows/ci.yml

```diff
@@ -89,7 +93,7 @@ jobs:
           password: ${{ secrets.QUAY_PASSWORD }}
       - uses: Alfresco/alfresco-build-tools/.github/actions/setup-java-build@v1.35.2
       - name: "Test application"
-        run: mvn ${{ env.MAVEN_CLI_OPTS }} clean verify -pl '!e2e-test'
+        run: mvn ${{ env.MAVEN_CLI_OPTS }} clean verify -pl '${{ matrix.subproject }}'
```

Comment:
Down side of this approach is: when someone will make changes in several modules, which depends in each other the test will fail, because only one module will be build within job, and changes from other modules won't be seen. This could be resolved by adding the `-am` flag, but this may result in building more then one module, depending on the dependencies list

Reply:
Good point! I'll add the flag and check if it will make jobs execution much slower...

Reply:
I think adding `-am` option is a good hint here, also `-DfailIfNoTests=false` could be useful if module has no tests at all (at least AFAIR)

Reply:
verify on distribution doesn't work for some reason so I extracted it to separate job